### PR TITLE
resolved Restrict the length of the API desription for API card

### LINKED ISSUE #2380: Restrict the length of the API desription for API card
# Goal
Make sure that the description of the API can fit into the API card. Some ideas:
* Set create a max length for API description
* If needed, have two descriptions. One (mandatory?) is for API card, one (optional) is for profile page. 

Having two different descriptions would allow the owner have an extensive descriptions of what the API can do for the user, but would need to come up with a sensible short description with only the main points that can be shown in the card

### PR DESCRIPTION
**Pull request by krashna-deligence**
Closes #2380 

## Changes
Describe your changes here as a bulleted list:
- Change max limit of description field

## Developer checklist
This checklist is to be completed by the PR developer:

- [ ] Alternative solutions were compared/discussed before writing code
    - [ ] trade-offs with this solution are considered acceptable
- [ ] Code in this PR adheres to the [project styleguide](CONTRIBUTING.md)
- [ ] This pull request does not decrease project test coverage
- [ ] If the code changes existing database collection(s), migration has been written
- [ ] If UI texts are added or changed, all texts are internationalized

## Reviewer checklist
Reviewed by: @username1

This list is to be completed by the pull request reviewer:
- [ ] Code works as described/expected
- [ ] Code seems to be error free
    - [ ] no browser console errors visible
    - [ ] no server console errors visible
    - [ ] passes CI build
- [ ] Code is written in a way that promotes maintainability
    - [ ] easy to understand
    - [ ] well organized
    - [ ] follows project coding standards and conventions
    - [ ] well documented

### CODE REVIEW

File: apinf_packages/apis/collection/schema.js [hunk 1]
```diff
@@ -21,7 +21,7 @@ Apis.schema = new SimpleSchema({
   },
   description: {
     type: String,
-    max: 1000,
+    max: 140,
```
Comment (marla-singer) [inline apinf_packages/apis/collection/schema.js]: Why did you do it?

Comment (krashna-deligence) [reply apinf_packages/apis/collection/schema.js]: Hi @marla-singer API consumers won't care about long descriptions telling people unnecessary details as per discussion in issue 

>+1 with the idea to have Max length for description field in the API profile.
>In Twitter, u need to express the most important things within a limitation of 140 characters per tweet.
>We should consider having similar feature in APInf as well.
>API consumers won't care about long descriptions telling people unnecessary details.
>Besides we also have external documentation link feature available.
>An API owner can always add a blog there describing the API in details.
>In Addition, there is OEMBD media content to give information related to the API.

Comment (marla-singer) [reply apinf_packages/apis/collection/schema.js]: But the issue description has the second point:
> If needed, have two descriptions. One (mandatory?) is for API card, one (optional) is for profile page.

And I don't see any approve/cancel this idea so I don't think that the task is completed

File: apinf_packages/apis/collection/schema.js [hunk 2]
```diff
@@ -20,6 +20,14 @@ Apis.schema = new SimpleSchema({
...
   },
   description: {
+    type: String,
+    max: 1000,
+    autoform: {
+      rows: 3,
+    },
+    optional: false,
+  },
+  description2: {
```
Comment (marla-singer) [inline apinf_packages/apis/collection/schema.js]: And why did you do that?

**Comment by preriasusi** [comment]
This isse #2380 is taken for icebox, not valid for APInf, not needed. Let's stop working with this.

**Comment by marla-singer** [comment]
Close it based on @preriasusi comment

**Pull request closed without merging**